TransferList/TransferList.js

```diff
@@ -71,6 +80,22 @@ const TransferListBase = kind({
 		 */
 		firstList: PropTypes.array,
 
+		/**
+		 * Sets the maximum number of items for the first list.
+		 *
+		 * @type {Number}
+		 * @public
+		 */
+		firstListMaximumCapacity: PropTypes.number,
```

Comment:
I suggest a minor abbreviation 'firstListMaxCapacity' and 'firstListMinCapacity'. Same for secondList.

Reply:
ok